gpl.py

```diff
@@ -0,0 +1,115 @@
+############## from https://github.com/trgao10/PuenteAlignment/blob/master/code/gplmk.m
+import numpy as np
+import using_kdtree
```

Comment:
What is using_kdtree? Is this strictly necessary? We can’t to be careful about extra packages not present in Slicer. Numpy and Scipy are ok, though.

Reply:
Same with sklearn, is this available in Slicer?